hexrd/imageseries/load/framecache.py

```diff
@@ -45,6 +63,29 @@ def _load_yml(self):
         self._meta = yamlmeta(d['meta'], path=self._cache)
 
     def _load_cache(self):
+        if self._style == 'fch5':
+            self._load_cache_fch5()
+        else:
+            self._load_cache_npz()
+
+    def _load_cache_fch5(self):
+        with h5py.File(self._fname, "r") as file:
+            if 'HEXRD_FRAMECACHE_VERSION' not in file.attrs.keys():
+                raise NotImplementedError("Unsupported file."
+                                          "HEXRD_FRAMECACHE_VERSION"
+                                          "is missing!")
```

Comment:
```suggestion
                raise NotImplementedError("Unsupported file. "
                                          "HEXRD_FRAMECACHE_VERSION "
                                          "is missing!")
```

Need to add spaces, otherwise the message will look like "Unsupported file.HEXRD_FRAMECACHE_VERSIONis missing!".